mne/tests/test_epochs.py

```diff
@@ -57,6 +59,70 @@
 rng = np.random.RandomState(42)
 
 
+def test_handle_duplicate_events():
+    """Test handling of duplicate events."""
+    # A general test case
+    EVENT_ID = {'aud': 1, 'vis': 2, 'foo': 3}
+    EVENTS = np.array([[0, 0, 1], [0, 0, 2],
+                       [3, 0, 2], [3, 0, 1],
+                       [5, 0, 2], [5, 0, 1], [5, 0, 3],
+                       [7, 0, 1]])
+
+    with pytest.raises(RuntimeError, match='Event time samples were not uniq'):
+        _handle_duplicate_events(EVENTS, EVENT_ID, event_repeated='error')
+
+    with pytest.raises(ValueError, match='`event_repeated` must be one of '):
+        _handle_duplicate_events(EVENTS, EVENT_ID, event_repeated='bogus')
+
+    events, event_id = _handle_duplicate_events(EVENTS, EVENT_ID, 'drop')
+    assert_array_equal(events, [[0, 0, 1], [3, 0, 2], [5, 0, 2], [7, 0, 1]])
+    assert event_id == {'aud': 1, 'vis': 2}
+
+    events, event_id = _handle_duplicate_events(EVENTS, EVENT_ID, 'merge')
+    assert_array_equal(events[0], events[1])
+    assert_array_equal(events, [[0, 0, 3], [3, 0, 2], ])
+    assert 'aud/vis' in event_id.keys()
```

Comment:
Isn't this line redundant with the one after?

Reply:
Yes! Will remove :+1: